src/components/atoms/GalleryImage/GalleryImage.story.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
```

Comment:
You're committing these changes (Gallery) to this PR. Gallery and Navigation Menu are different components and each one needs to have their own PR.  I can approve it, but at work, this PR will be rejected.